core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -164,6 +164,7 @@ public class PropServerConfiguration implements ServerConfiguration {
     private final int maxUncommittedRows;
     private final MetricsConfiguration metricsConfiguration = new PropMetricsConfiguration();
     private final boolean metricsEnabled;
+    private final MicrosecondClock microsecondClock;
```

Comment:
@puzpuzpuz I did that. But it reformat some other places too for example this MicrosecondClock got repositioned upon save.

Reply:
That's ok. Occasionally, we get patches that aren't properly auto-formatted.